website/src/content/blog/biome-v1-6.mdx

```diff
@@ -0,0 +1,273 @@
+
+```json name="shared-configs/package.json"
+{
+  "name": "@shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.json"
+  }
+}
+```
+
+This set up allows to expose a specifier `@shared-configs/biome`, which you use inside your `biome.json` file.
+
+```json name="project/biome.json"
+{
+  "extends": ["@shared-configs/biome"]
+}
+```
+
+The resolution of the dependencies is powered by the library [`oxc-resolver`](https://github.com/oxc-project/oxc-resolver), one of the many libraries provided by the [OXC project](https://oxc-project.github.io/). It's battle-tested and spec compliant!
+
+:::note
+You can also export `biome.jsonc` files!
+
+```json name="shared-configs/package.json" ins="./biome.jsonc"
+{
+  "name": "@shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.jsonc"
+  }
+}
+```
+:::
+
+### Reduced memory footprint
+
+We **reduced** the size our configuration **by a factor of 6.5**! This change might not have massive effects on the speed of the program, but it greatly reduced the memory used when running the CLI or the LSP.
+
+## New formatting options
+
+Other than fixes, the formatter provides two new options that should improve the compatibility with Prettier.
+
+### Option `attributePosition`
+
+The contributor [@octoshikari](https://github.com/octoshikari) implemented this new feature by themselves! Huge thank you for helping the Biome project.
+
+When `formatter.attributePosition` has the value `multiline`, all attributes of HTML-ish languages (JSX/TSX as for time of writing) will be collapsed on multiple lines regardless of their numbers:
+
+
+<div class="card">
+    <div class="card-column">
+        <h4>With variant `auto` (default)</h4>
+        The attributes are automatically formatted, and they will collapse in multiple lines only when they hit certain criteria.
+
+        ```jsx title="file.jsx"
+        <Button as="link" style="primary" href="https://example.com">
+            Hit me
+        </Button>
+        ```
+    </div>
+    <div class="card-column">
+        <h4>With variant `multiline`</h4>
+        The attributes are always formatted on multiple lines, regardless.
+
+        ```jsx title="file.jsx"
+        <Button
+            as="link"
+            style="primary"
+            href="https://example.com"
+        >
+            Hit me
+        </Button>
+        ```
+    </div>
+</div>
+
+
+### Option `json.formatter.trailingCommas`
+
+Previously, Biome parser introduced an option that would allow to parse JSON and JSONC files that contained a trailing comma. This was required to ease the friction caused by other tools that
+tolerate trailing commas by default (e.g. VSCode, Prettier, etc.).
+
+Unfortunately, our formatter wasn't as tolerant. But with this release, we've introduced the option `json.formatter.trailingCommas`. It allows you to apply the same rules as with `js.formatter.trailingComma`.
```

Comment:
Are there some rationals between the use of a plural form for _json_ and singular form for _js_?

Reply:
Singular was a mistake, plural is the way to go. Other options use the plural. We were superficial when we introduced the trailing comma option for JavaScript